source/Halibut/Transport/Protocol/WebSocketStream.cs

```diff
@@ -50,7 +50,11 @@ public async Task<string> ReadTextMessage()
             var buffer = new ArraySegment<byte>(new byte[10000]);
             while (true)
             {
+                // TODO - ASYNC ME UP!
+                // What should the timeout be here? thew new code that allows passing in a timeout or the static value?
```

Comment:
https://github.com/OctopusDeploy/Halibut/pull/414 changes this stream to add async methods